book/accessibility.md

```diff
@@ -955,6 +955,11 @@ class Tab:
                 self.set_needs_render()
                 return
             elt = elt.parent
+
+    def keypress(self, char):
+        if self.focus and self.focus.tag == "input":
+            if not "value" in self.focus.attributes:
+                self.activate_element(self.focus)
```

Comment:
Just so I understand the bug—is it that tabbing into something and then typing into it would crash (because the `value` attribute didn't exist?).

Reply:
Yes, exactly.